powerauth-webflow-authentication/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/controller/AuthMethodController.java

```diff
@@ -259,7 +259,7 @@ protected UpdateOperationResponse authorize(String operationId, String userId, S
     protected UpdateOperationResponse failAuthorization(String operationId, String userId, List<KeyValueParameter> params) throws NextStepServiceException, AuthStepException {
         logger.info("Fail step started, operation ID: {}, user ID: {}, authentication method: {}", operationId, userId, getAuthMethodName().toString());
         // validate operation before requesting update
-        GetOperationDetailResponse operation = validateOperationState(operationId);
+        GetOperationDetailResponse operation = getOperation(operationId);
```

Comment:
Is this really OK change?

Reply:
Yes, the original method was duplicate and actually didn't have all functionality of getOperation().